samples/bluetooth/direction_finding_central/README.rst

```diff
@@ -1,12 +1,13 @@
-.. _bluetooth_direction_finding_central:
+.. zephyr:code-sample:: bluetooth_direction_finding_central
+   :name: Direction Finding Central
+   :relevant-api: bluetooth
```

Comment:
What was your process for finding the relevant APIs?

I think any `central` sample should also have `bt_conn` as a API reference, as that is used to establish the connections

Reply:
My process can be best described as "best effort" for an initial pass :)

Reply:
Alright - Thought you might have made a script to check which functions were called (which in theory is possible, although not a 5-min task ;) )